Tools/Scripts/webkitpy/benchmark_runner/browser_driver/osx_minibrowser_driver.py

```diff
@@ -38,7 +38,6 @@ def launch_url(self, url, options, browser_build_path, browser_path):
             env['WEBKIT_SIGNPOSTS_ENABLED'] = '1'
             env['__XPC_WEBKIT_SIGNPOSTS_ENABLED'] = '1'
             env['__XPC_JSC_exposeProfilersOnGlobalObject'] = '1'
-            env['__XPC_WebKitPerformanceSignpostEnabled'] = '1'
```

Comment:
Even if we agreed on removing `__XPC_WebKitPerformanceSignpostEnabled`, for backward compatibility, we cannot remove this from the harness right away. And we probably want to keep it backward compatibility at least for a year.